web/styles/dark_theme.css

```diff
@@ -81,6 +81,12 @@
         }
     }
 
+    .topics_popover {
+        .topic-name {
+            color: hsl(236deg 33% 90%);
```

Comment:
While this solution may work for the specific element, it does not address the issue for other elements. If we do not require a custom color for this particular element, then we should investigate the root cause of the problem.

After some research, I discovered that the main branch utilizes the color from `.dark-theme:root body`. However, due to the use of `tippy` and its default theming, the colors are currently broken. Perhaps we should define this color for additional elements within the dark theme?

What are your thoughts?

Reply:
In the case of dark theme, the CSS for Tippy popovers is defined considering all the text as anchor tags. However, the topic name is a `<p>` tag which resulted in broken colors. However, I have fixed this by adding `<p>` in the Tippy CSS. Let me know if you want me to make any other changes.